### ISSUE TITLE
Should we scrape an issue that only has one comment?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/github-scraper#24]
Should we scrape issues that don't have any comments?
e.g:
![github-issue-has-more-than-one-comment](https://cloud.githubusercontent.com/assets/194400/8946136/e00eff7e-3585-11e5-87e2-b61a39dc8aea.png)

**Comment by iteles** [comment]
Yes.

When there are _'no comments'_ what that actually means is that there is still an issue _description_, just no interaction on that issue yet.

_(This is the 'no comments' issue highlighted above)_
<img width="908" alt="no-comments-issue-actually-has-description" src="https://cloud.githubusercontent.com/assets/4185328/8946220/a59d414c-3586-11e5-8725-d3413bd8e433.png">

The issue information is still very valuable, especially for https://github.com/dwyl/tudo functionality.

**Comment by nelsonic** [comment]
Right, so we still need to scrape it just to extract the body of the issue? :question:

**Comment by iteles** [comment]
I reckon so, as I assume we can get all the rest of the info we need (assignee, labels, etc) from the issue list view.

**Comment by nelsonic** [comment]
Already got most of the meta info (including *milestone*, as discussed ... or "_milesontes_" as my commit called them... :stuck_out_tongue_winking_eye: )
but we will need to extract the *participants* from the issue page ...
